diff --git a/utilities/wireshark_dissectors/openflow/packet-openflow.c b/utilities/wireshark_dissectors/openflow/packet-openflow.c
index aea00f8..8d98335 100644
--- a/utilities/wireshark_dissectors/openflow/packet-openflow.c
+++ b/utilities/wireshark_dissectors/openflow/packet-openflow.c
@@ -766,7 +766,7 @@ static gint ett_ofp_error_msg_data = -1;
 void proto_reg_handoff_openflow()
 {
     openflow_handle = create_dissector_handle(dissect_openflow, proto_openflow);
-    dissector_add(TCP_PORT_FILTER, global_openflow_proto, openflow_handle);
+    dissector_add_uint(TCP_PORT_FILTER, global_openflow_proto, openflow_handle);
 }
 
 #define NO_STRINGS NULL
