--- features/example.message.hello.feature	2012-12-20 14:49:33.546100003 +0900
+++ /home/gakusei/project/trema/features/example.message.hello.feature	2012-12-20 14:39:25.974100000 +0900
@@ -4,22 +4,29 @@
   I want to send hello messages to openflow switches
   So that I can start transactions with switches
 
-  @wip
+
   Scenario: Hello trema
     When I try trema run "./objects/examples/openflow_message/hello 10" with following configuration (backgrounded):
       """
-      vswitch("hello") { datapath_id "0xabc" }
+      custom_switch("hello") { 
+        datapath_id "0xabc" 
+        path "./objects/examples/openflow_switch/hello_switch"
+      }
       """
       And wait until "hello" is up
       And I terminated all trema services
-    Then the log file "openflowd.hello.log" should include "received: OFPT_HELLO" x 11
+    Then the log file "customswitch.hello.log" should include "received: OFPT_HELLO" x 11
+
 
   @wip
   Scenario: Hello trema in Ruby
     When I try trema run "./src/examples/openflow_message/hello.rb 0xabc, 10" with following configuration (backgrounded):
       """
-      vswitch("hello-r") { datapath_id "0xabc" }
+      custom_switch("hello-r") {
+        datapath_id "0xabc"
+        path "./objects/examples/openflow_switch/hello_switch"
+      }
       """
       And wait until "HelloController" is up
       And I terminated all trema services
-    Then the log file "openflowd.hello-r.log" should include "received: OFPT_HELLO" x 11
+    Then the log file "customswitch.hello-r.log" should include "received: OFPT_HELLO" x 11
